docs/installing/storage.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: page
+title: File Storage
+---
+
+# File Storage
+
+<p class="lead">
+  Storage for raw emails and attachments.
+</p>
+
+<div class="attention-box">
+  In Alaveteli 0.41 we switched from using our own bespoke storage for raw
+  emails and attachments to use a third party library called
+  <a href="{{ page.baseurl }}/docs/glossary/#activestorage" class="glossary__link">Active Storage</a>.
+  This allows greater flexibility in where you store files.
+</div>
+
+The default approach to storing files is to store on the local disk but as your
+install because larger this becomes less desirable as it means a lot of data to
```

Comment:
```diff
- install because larger this becomes less desirable as it means a lot of data to
+ install becomes larger this becomes less desirable as it means a lot of data must
```